src/constant/index.js

```diff
@@ -1,9 +1,10 @@
 export const VERSIONS_GROUPS = {
   DEFAULT: {
-    STABLE_VERSION: "3.1",
+    STABLE_VERSION: "3.2",
     LEGACY_VERSION: "2.0",
```

Comment:
```suggestion
    LEGACY_VERSIONS: ["2.0", "3.1"],
```

See other comments related to it